src/DotNetty.Buffers/AbstractByteBuffer.cs

```diff
@@ -302,6 +303,12 @@ public virtual long GetLong(int index)
 
         public virtual char GetChar(int index) => Convert.ToChar(this.GetShort(index));
 
+        public virtual float GetFloat(int index)
+        {
+            int raw = this.GetInt(index);
+            return Unsafe.As<int, float>(ref raw);
```

Comment:
Then there will be no need for adding package

---
In reply to: [116194501](https://github.com/Azure/DotNetty/pull/209#discussion_r116194501) [](ancestors = 116194501)